test/test_tensor_creation_ops.py

```diff
@@ -2668,6 +2668,15 @@ def test_empty_tensor_props(self, device):
             y = torch.empty(tuple(size_ones_instead_of_zeros), device=device)
             self.assertEqual(x.stride(), y.stride())
 
+    @onlyNativeDeviceTypes
+    def test_empty_overflow(self, device):
+        with self.assertRaisesRegex(RuntimeError, 'Storage size calculation overflowed'):
+            torch.empty([2, 4, 536870912, 536870912], dtype=torch.float64)
+        with self.assertRaisesRegex(RuntimeError, 'Storage size calculation overflowed'):
+            torch.empty([8, 8, 536870912, 536870912], dtype=torch.float64)
+        with self.assertRaisesRegex(RuntimeError, 'Storage size calculation overflowed'):
+            torch.empty_strided([8, 8], [8 * 536870912**2, 1], dtype=torch.float64)
```

Comment:
Same as above, `8 * 536870912**2` is somewhat hard to read
```suggestion
            torch.empty_strided([8, 8], [2**61, 1], dtype=torch.float64)
```